internal/services/mysql/mysql_flexible_server_resource_test.go

```diff
@@ -568,6 +568,35 @@ func TestAccMySqlFlexibleServer_updateToWriteOnlyPassword(t *testing.T) {
 	})
 }
 
+func TestAccMySqlFlexibleServer_updatePublicNetworkAccess(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_mysql_flexible_server", "test")
+	r := MySqlFlexibleServerResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.publicNetworkAccess(data, "null"),
```

Comment:
We normally don't test null but we might want to so I'm curious what are you hoping to see out of this test step?

Reply:
Testing the default behavior. The Azure REST API has the public access enabled as optional, so it doesn't need to be sent, so this is just to ensure we don't have one of those Computed / Optional situations I like had before, where when not specified, the first plan after the initial create wants to change the setting to the zero value.

Reply:
Yea...so I think since Azure defaults public_network_access enabled to true, we can also default it to true and then don't have to test null